spark-testing/spark_utils.py

```diff
@@ -217,20 +217,17 @@ def spark_security_session():
     service_account = SPARK_SERVICE_ACCOUNT
     secret = SPARK_SERVICE_ACCOUNT_SECRET
 
-    def grant_driver_permission(service_account_name, app_id):
-        dcosurl, headers = sdk_security.get_dcos_credentials()
+    def grant_driver_permission(service_account_name, service_name):
+        app_id = "/{}".format(service_name)
```

Comment:
Instead of removing the `/` from `FOLDERED_SERVICE_NAME`, let's use:
```python
app_id = "/{}".format(service_name.lstrip("/"))
```
to ensure that `app_id` only starts with a single slash.

Reply:
👍